tests/cases/map.js

```diff
@@ -849,15 +843,15 @@ describe('geo.core.map', function () {
 
   describe('Internal methods', function () {
     it('resizeSelf', function () {
-      var m = create_map();
-      expect(m.size()).toEqual({width: 500, height: 500});
-      $('#map-create-map').css({width: '400px', height: '400px'});
-      expect(m.size()).toEqual({width: 500, height: 500});
+      var m = createMap();
+      expect(m.size()).toEqual({width: 640, height: 360});
```

Comment:
why change the size here?

Reply:
I'm using a generic test element that is 640 x 360 (for no particular reason that I know of).  Before, this create a map that was a different size; since we don't really care what the size is (only that it is what we expect), when calling the generic function, test that it is the size we expect.

Reply:
ah okay. thanks.